app/ui.R

```diff
@@ -86,22 +87,24 @@ ui <- fluidPage(
             )
           )
         ),
-          tabPanel("Run Clustering",
-            markdown(paste0(
-              'Clustering is applied across all pigment samples to ',
-              'differentiate between samples taken under different conditions. ',
-              'A "dynamic tree cut" algorithm is applied to generate the tree.'
-            )),
-            quartoReportUI("cluster",
-              defaultSetupCode = paste(
-                "inputFile <- 'pigments.rds'",
-                "outputFile <- 'clusters.rds'",
-                "minClusterSize <- 14",
-                sep="\n"
-              )
-            )
-            # TODO: download clusters .csv
+        tabPanel("Run Clustering",
```

Comment:
Should we consider reformatting to use this kind of alignment throughout the project?

I'm unfamiliar with R coding standards and automated formatters (like PEP8 and flake8 for python).

We should research and discuss further.